src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/DfsTests.cs

```diff
@@ -92,47 +95,45 @@ public async Task DFS_should_add_and_read_binary()
                 1, 2, 3
             };
             var ms = new MemoryStream(binary);
-            var dfs = new Dfs(_ipfs, _hashProvider, _logger);
-            var id = await dfs.AddAsync(ms, "", cts.Token);
-            using (var stream = await dfs.ReadAsync(id, cts.Token))
+            
+            // var dfs = new Dfs(_ipfs, _hashProvider, _logger);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)